docs/custom-nodes.md

```diff
@@ -122,11 +125,9 @@ The following sections provide some common patterns for implementing custom node
 Nodes that perform an operation but return the input as the output.
 
 ```kotlin
-fun <T> AIAgentSubgraphBuilder<*, *>.loggingNode(
-    name: String? = null
-): AIAgentNodeDelegateBase<T, T> = node(name) { input ->
+val loggingNode by node<T, T>("node_name") { input ->
```

Comment:
Instead of `<T, T>` there should be concrete types. Let's make it `<String, String>`